bigquery/unique_identifiers.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Unique Identifiers
+nav_order: 2
+parent: BigQuery
+---
+
+# Unique Identifiers (DCid)
```

Comment:
DCID or dcid ?

(I don't think stylize it as DCid elsewhere)

Reply:
We've used DCID elsewhere on this site, and dcid on www. Can we standardize to lowercase?

Reply:
done